samples/main.js

```diff
@@ -92,3 +103,7 @@ new Vue( {
 	el: '#app',
 	router: router
 } );
+
+function scriptLoaded( namespace ) {
+	console.log( 'CKEDITOR version: ', namespace.version );
+}
```

Comment:
Those lines seem to be a leftover before introducing `defaultMixin`, right?

Reply:
Yup, I forgot about it.